### PR TITLE
Add a new requisitions tab and make phoron obtainable.

### PR DESCRIPTION
**Pull request by Zarq0**
## About the PR

Added a new requisitions tab, Research and made phoron obtainable through requisitions.

## Why / Balance

Made phoron obtainable through requisitions so the bars can be ordered and grinded for the Phoron reagent, it is needed to make some medicinal reagents.

## Media

https://github.com/RMC-14/RMC-14/assets/75623951/1aac0567-2faa-4f75-a67f-d7ae3c020f56

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added a new requisition tab, Research.
- tweak: Made phoron obtainable through requisitions.

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Structures/Storage/crates.yml [hunk 1]
```diff
@@ -319,6 +319,25 @@
...
+    - state: base
+      map: ["enum.StorageVisualLayers.Base"]
+    - state: closed
+      map: ["enum.StorageVisualLayers.Door"]
+    - state: welded
+      visible: false
+      map: ["enum.WeldableLayers.BaseWelded"]
+  - type: Icon
+    sprite: _RMC14/Structures/Storage/Crates/phoron.rsi
+
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Storage/crates.yml:340]:
```suggestion
  parent: CMCrateSecure
  id: RMCCratePhoron
  name: phoron crate
  description: A phoron crate.
  components:
  - type: AccessReader
    access: [["CMAccessResearch"],["CMAccessEngineering"]]

```
Drop the new RSI, 13 uses the secure crate, and you'd need access locks sprites, and a place to put the req paperwork, so it's overall better to use `CMCrateSecure` as a parent anyways so you don't need to create anything new.

Make sure to test these changes after though

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
TLDR
- use `CMCrateSecure` as a parent for the empty crate
- drop the new rsi and sprites
- make a 30 stack of phoron for the crate

**Comment by Zarq0** [comment]
Done,  made the changes requested, tested and they seem to work for me.

**Comment by Zarq0** [comment]
https://github.com/RMC-14/RMC-14/assets/75623951/fa8a6781-8488-4a53-bde9-8b6dacf938df

**Review by QuietlyWhisper — APPROVED** [review]
Seems to be fine thanks!
